### PR TITLE
Reenable tests and forward fixture logs to test logs

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

**Comment by jkotalik** [comment]
Assuming this is for debugging the failure?

**Comment by pakrym** [comment]
Yep

File: test/IISExpress.FunctionalTests/InProcess/FixtureLoggedTest.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+using Microsoft.Extensions.Logging.Testing;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
+{
+    public class FixtureLoggedTest: LoggedTest
+    {
+        private readonly IISTestSiteFixture _fixture;
+
+        public FixtureLoggedTest(IISTestSiteFixture fixture)
```
Comment (jkotalik) [inline test/IISExpress.FunctionalTests/InProcess/FixtureLoggedTest.cs]: Why not make the rest of the fixture tests inherit FixtureLoggedTest?

File: test/Common.FunctionalTests/Inprocess/SynchronousReadAndWriteTests.cs [hunk 2]
```diff
@@ -6,23 +6,22 @@
...
 namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
 {
     [Collection(IISTestSiteCollection.Name)]
-    public class SynchronousReadAndWriteTests
+    public class SynchronousReadAndWriteTests: FixtureLoggedTest
     {
         private readonly IISTestSiteFixture _fixture;
 
-        public SynchronousReadAndWriteTests(IISTestSiteFixture fixture)
+        public SynchronousReadAndWriteTests(IISTestSiteFixture fixture):base(fixture)
```
Comment (jkotalik) [inline test/Common.FunctionalTests/Inprocess/SynchronousReadAndWriteTests.cs]: nit formatting on these lines.

File: test/Common.FunctionalTests/Utilities/IISTestSiteFixture.cs [hunk 3]
```diff
@@ -60,5 +66,88 @@ public void Dispose()
...
+            private readonly List<ForwardingLogger> _loggers = new List<ForwardingLogger>();
+
+            private ILoggerFactory _loggerFactory;
+
+            public ILoggerFactory LoggerFactory
+            {
+                get => _loggerFactory;
+                set
+                {
+
```
Comment (jkotalik) [inline test/Common.FunctionalTests/Utilities/IISTestSiteFixture.cs]: nit newline.

File: test/IISExpress.FunctionalTests/InProcess/FixtureLoggedTest.cs (hunk 1 above)
Comment (pakrym) [reply test/IISExpress.FunctionalTests/InProcess/FixtureLoggedTest.cs]: I'd like to ease it in. See if it breaks things

**Comment by jkotalik** [comment]
C:\projects\iisintegration\test\Common.FunctionalTests\Inprocess\SynchronousReadAndWriteTests.cs(15,48): error CS0246: The type or namespace name 'FixtureLoggedTest' could not be found (are you missing a using directive or an assembly reference?) [C:\projects\iisintegration\test\IIS.FunctionalTests\IIS.FunctionalTests.csproj]

**Comment by pakrym** [comment]
cc @JunTaoLuo parts of this PR (attaching current test logger to fixture that has it's own logging) would be nice to have in Testing.Logging eventually.

**Pull request merged**